### PR TITLE
Add tinkerbell add user example

### PR DESCRIPTION
**Pull request by chrisdoherty4**
Add an examples directory that can be used to point customers at useful snippets.

### CODE REVIEW

File: examples/tinkerbell/adding-a-user-to-tinkerbelltemplateconfiguration.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+spec:
+  template:
+    global_timeout: 6000
+    id: ""
+    name: cpd
+    tasks:
+    - actions:
+      // Append to existing actions.
+      - environment:
+          BLOCK_DEVICE: /dev/sda2
```
Comment (ptrivedi) [inline examples/tinkerbell/adding-a-user-to-tinkerbelltemplateconfiguration.md:19]: put a comment next to this saying they would need to change BLOCK_DEVICE based on their disk type. Maybe should also mention that our Ubuntu images have UEFI as the first partition, so OS would be on the second.

Comment (chrisdoherty4) [reply examples/tinkerbell/adding-a-user-to-tinkerbelltemplateconfiguration.md]: Updated

File: examples/tinkerbell/adding-a-user-to-tinkerbelltemplateconfiguration.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          FS_TYPE: ext4
+          CHROOT: y
+          DEFAULT_INTERPRETER: "/bin/sh -c"
+          CMD_LINE: "useradd --password $(openssl passwd -1 tinkerbell) --shell /bin/bash --create-home --groups sudo tinkerbell"
+        image: public.ecr.aws/l0g8r8j6/tinkerbell/hub/cexec:6c0f0d437bde2c836d90b000312c8b25fa1b65e1-eks-a-v0.0.0-dev-build.2301
+        name: "create-user"
+        timeout: 90
```
[the remark this answers is not in the document]
Comment (chrisdoherty4) [reply examples/tinkerbell/adding-a-user-to-tinkerbelltemplateconfiguration.md]: Updated

**Review by abhinavmpandey08 — APPROVED** [review]
/lgtm

**Pull request merged**